package.json

```
@@ -1,28 +1,12 @@
 {
-  "name": "my-app",
-  "version": "0.1.0",
+  "name": "react-template",
+  "version": "2.0.0",
```

```suggestion
  "name": "my-app",
  "version": "0.0.0",
```
Those are metadata for the application that will use the template. We should not pollute those with nonsense like the version already being the 2.0.